qemu/overlay/etc/zinit/init/bootstrap.sh

```diff
@@ -0,0 +1 @@
+../../../../../bootstrap/etc/zinit/init/bootstrap.sh
```

Comment:
this path is out of the repo no ?

Reply:
No, the `bootstrap` directory is at the root of the repo